External/Plugins/ASCompletion/Completion/ASComplete.cs

```diff
@@ -3496,22 +3497,18 @@ private static ASExpr GetExpression(ScintillaControl sci, int position, bool ign
                         expression.Separator = ';';
                         break;
                     }
-                    else if (hasGenerics && (genCount > 0 || c == '<'))
+                    else if (hasGenerics && c == '<')
                     {
-                        if (c == '<')
+                        sbSub.Insert(0, c);
+                        if (genCount < 0
+                            && sci.ConfigurationLanguage == "as3"
+                            && position > minPos && sci.CharAt(position - 1) != '.')
                         {
-                            sbSub.Insert(0, c);
-                            if (genCount < 0
-                                && sci.ConfigurationLanguage == "as3"
-                                && position > minPos && sci.CharAt(position - 1) != '.')
-                            {
-                                position--;
-                                expression.Separator = ' ';
-                                break;
-                            }
-                            genCount--;
+                            position--;
+                            expression.Separator = ' ';
+                            break;
                         }
-                        else sb.Insert(0, c);
+                        genCount--;
```

Comment:
Sorry, Apparently accidentally got into this commit, but in any case, in the construction ```else if (hasGenerics && (genCount> 0 || c == '<'))``` genCount can not be greater than zero